RespiratoryAssistDevices/R4/resources/Questionnaire-R4-RespiratoryAssistDevicesFaceToFace.json

```diff
@@ -0,0 +1,1475 @@
+                "language": "text/cql",
+                "expression": "\"RespiratoryAssistDevicesFaceToFacePrepopulation\".EncounterDate"
+              }
+            }
+          ],
+          "type": "date"
+        },
+        {
+          "linkId": "5.5",
+          "text": "Was study performed in a facility?",
+          "type": "choice",
+          "answerOption": [
+            {
+              "valueCoding": {
+                "code": "Y",
+                "display": "Yes"
+              }
+            },
+            {
+              "valueCoding": {
+                "code": "N",
+                "display": "No"
+              }
+            }
+          ],
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"BasicClinicalInfoPrepopulation\".AnswerYes"
+              }
+            },
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+              "valueCodeableConcept": {
+                "coding": [
+                  {
+                    "code": "radio-button",
+                    "system": "http://hl7.org/fhir/questionnaire-item-control"
+                  }
+                ]
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "linkId": "6",
+      "text": "Other qualifying observations",
+      "type": "group",
+      "required": true,
+      "item": [
+        {
+          "linkId": "6.1",
+          "text": "If patient has a neuromuscular disease only:",
+          "type": "boolean",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"RespiratoryAssistDevicesFaceToFacePrepopulation\".HasNeuromuscularDisease"
+              }
+            }
+          ],
+          "item": [
+            {
+              "linkId": "6.1.1",
+              "text": "Maximal inspiratory pressure in units of cm H20:",
+              "type": "quantity",
+              "required": true,
+              "extension": [
+                {
+                  "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+                  "valueExpression": {
+                    "language": "text/cql",
+                    "expression": "\"RespiratoryAssistDevicesFaceToFacePrepopulation\".MIP"
+                  }
+                }
+              ],
+              "enableWhen": [
+                {
+                  "question": "6.1",
+                  "operator": "=",
+                  "answerBoolean": true
+                }
+              ]
+            },
+            {
+              "linkId": "6.1.2",
+              "text": "Forced vital capacity in % predicted:",
+              "type": "quantity",
```

Comment:
@yunwwang 
Do we want to add extension for this kind of display to limit the value as integer or decimal only? The UI provides some validation.  Or we should use the type "quantity"? It only displays the value portion of the Quantity if we use "quantity" as type. 
 {
                      "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-unit",
                      "valueCoding": {
                        "display": "%"
                      }
                    }

Reply:
Don't use quantity

Reply:
In the case of displaying the Neck Circumference value, I was not able to load the questionnaire successfully when I used type:decimal. I tried different ways and used same code from other ruleset but never got success. Not wanting to spend time to debug the problem, I just stayed with using type:quantity which works fine for my ruleset.

Reply:
We are not using quantity in table and it seems quantity works well here. Also, we don't really need to pre-populate physical exam section.

Reply:
@cwanmtr I think another benefits to use the extension of questinnaire-unit is that the unit is automatically shown on the UI. Please see the screen shot below. With "quantity", we don't have the benefit. 

![Screen Shot 2020-07-31 at 8 53 05 AM](https://user-images.githubusercontent.com/58572522/89036907-92d8bc80-d30b-11ea-99ce-e52fc08c4551.png)


In addition, we don't need to pre-populate the fields of physical exam etc.